### PR TITLE
ok? This is how I needed to do?

### PR DESCRIPTION
**Pull request by Nehakanki**

### CODE REVIEW

**Comment by Nehakanki** [comment]
Only SignUp remaining

File: Controller/auth_logic.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        const existing_email = await User.findOne({ email });
+        if(existing_email){
+            return res.status(400)
+            .json({
+                success:"false",
+                message :"User aldready Exist , Go for Login Process"
+            })
+        }
+        //There exist new User , Hash the password and then enter into DB (using 10 salt round --> optimal one)
+        const hashedPassword = await bcrypt.hash(password, 10);
```
Comment (Krutik30) [inline Controller/auth_logic.js:29]: seed jesa kuchh nai atta isme?

File: Controller/auth_logic.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+exports.Login = async (req, res)=>{
+    const {name, email, password} = req.body;
+     //1. validate the required fields
+     if(!name || !email || !password ){
+        return res.status(400)
+        .json({
+            success:"false",
+            message :"Enter all the required field"
+        })
```
Comment (Krutik30) [inline Controller/auth_logic.js:83]: here checking token first or 
just ese hi

ya tum usse direct home pe hi land karvati ho?

**Review by Krutik30 — APPROVED** [review]
No need to push pkg-lock 
gitingore or delete it always

**Pull request merged**